docs/source/conf.py

```diff
@@ -196,7 +196,7 @@
 # If true, "Created using Sphinx" is shown in the HTML footer. Default is True.
 html_show_sphinx = False
 
-html_add_permalinks = " "
+html_permalinks = True
```

Comment:
Fix for the following warning:

```
WARNING: html_add_permalinks has been deprecated since v3.5.0. Please use html_permalinks and html_permalinks_icon instead.
```